vars/ohpc-1.3

```diff
@@ -0,0 +1,19 @@
+---
+# OpenHPC 1.3 on CentOS 7
```

Comment:
I've just run a version of this branch on a base Centos**7** image with all the "@" removed and it worked fine. So maybe we don't actually need to differentiate between v1 and v2?

Reply:
However I like the package list setup - much tidier/faster than it was before.

Reply:
Good to know.  We still need the differences, but only for munge now.